src/containers/my-quizzes/create-or-edit-quiz-container/CreateOrEditQuizContainer.tsx

```diff
@@ -198,6 +199,18 @@ const CreateOrEditQuizContainer = ({
         })
       : void addNewQuiz({ title, description, items: questions, category })
 
+  const createNewQuestionButtonInactive = (
+    <Tooltip
+      placement='top'
+      title={t('myResourcesPage.quizzes.savePreviousQuestion')}
```

Comment:
I suggest adding one more prop to tooltip called `arrow` to make tooltip more appealing

![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/109922101/7d18d964-b7c6-4584-992e-db054784c4c0)

You can dive into docs if you need more details on how to use it: https://mui.com/material-ui/react-tooltip/

Reply:
Thanks! Added.